.github/workflows/push.yml

```diff
@@ -39,3 +39,15 @@ jobs:
                 php-version: 7.3
                 coverage: none # disable xdebug, pcov
           - run: composer install --no-progress && composer rector
+
+  fatal-scan:
+      name: Scan Fatal Errors
+      runs-on: ubuntu-latest
+      steps:
+          - uses: actions/checkout@v2
+          - name: Setup PHP
+            uses: shivammathur/setup-php@v1
+            with:
+                php-version: 7.3
+                coverage: none # disable xdebug, pcov
+          - run: composer install --no-progress && bin/rector scan-fatal-errors tests/Source/FatalErrors
```

Comment:
&& should never be used, it's not clear if the first fails && the second.

Could you move it to generic install section above somehow? The same way other CI's have it?

Reply:
Done